pkg/reader/ccip.go

```diff
@@ -585,6 +586,7 @@ func (r *ccipChainReader) GetExpectedNextSequenceNumber(
 			sourceChainSelector, r.destChain)
 	}
 
+	r.lggr.Debugw("chain reader returning expected next sequence number", "seqNum", expectedNextSequenceNumber)
```

Comment:
Please create a logger like the other methods,

```golang
lggr := logutil.WithContextValues(ctx, r.lggr)
```

So that we have the ocrSeqNr and ocrPhase in the log fields.